source/src/localisation/cs_MeshTopology.cpp

```diff
@@ -10,23 +10,25 @@
 
 #include <cmath>
 
-// REVIEW: This won't be recognized by binary logger.
-#define RSSIDATATRACKER_LOGd LOGd
-#define RSSIDATATRACKER_LOGv LOGnone
+#define LOGMeshTopologyDebug LOGd
```

Comment:
If you want to have `LOGMeshTopologyDebug` to log by default, then just use `LOGd` in the code. These defines should be extra logs for when specifically debugging this class, and thus be `LOGnone` by default.

Reply:
I'm not sure if I agree with this logging style paradigm. The defines help categorize the logging statements in the file and make it easier to turn off logging of things that may be too verbose when debugging problems in other files. Can we discuss that?

Reply:
Agree with @ArrowAcrobatics. Currently, setting `SERIAL_VERBOSITY` to `SERIAL_DEBUG` creates a binary that is too large to fit (when not using "binary logging") over UART. There must be a mechanism in place to do something fine-grained per compilation unit. The Nordic code has macros for this per driver / module. It is ugly though and generates loads of compiler flags. I'd recommend the use of `include/cfg/cs_StaticConfig.h.in` for this. Those values are known at compile time and can be **const**.